docs/architecture/baking.md

```diff
@@ -5,22 +5,25 @@ last_update:
   date: 21 February 2024
 ---
 
-Baking is the process of creating new blocks.
-Bakers are nodes that validate pending operations, package them into a block, sign the block, publish the new block to other nodes, and verify that the blocks that other nodes publish are valid.
-The protocol rewards bakers with tez for baking blocks and for attesting that other blocks are valid.
+Baking is the process of creating new blocks in the Tezos blockchain.
+Bakers are daemons complementary to Tezos nodes that cooperate to achieve consensus about the next block to add.
+For doing so, bakers validate pending operations, package them into a block, sign the block, propose the new block to other nodes, and verify that the blocks that other bakers propose are valid.
+
+Baker daemons are run on behalf of user accounts which stake tez to guarantee honest participation and receive rewards for their participation.
+By extension, bakers also denote the users running baker daemons on behalf of their user accounts.
```

Comment:
Not sure what this means. Maybe "Each baker daemon is connected to a single user account."?